recipes/rhash/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "rhash" %}
+{% set version = "1.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rhash/RHash/archive/v{{ version }}.tar.gz
+  sha256: 1f6daa0c066c94d2575f0aace76f34e5800c51f59f4b30029ddcfa9799564f98
+  patches:
+    # https://searchcode.com/file/92041882/Library/Formula/rhash.rb
```

Comment:
So the patch is from Homebrew (linked below)? Has it been submitted upstream? Also what is the purpose of this patch? Please include a summary of it in the comment above.

ref: https://github.com/Homebrew/homebrew-core/blob/87e318ab9f1f8268dc3ae446f7573fcda3ae6f4c/Formula/rhash.rb#L36-L57